ftp-native/src/main/java/org/ballerinalang/stdlib/ftp/util/FtpUtil.java

```diff
@@ -127,6 +127,9 @@ private static String createUrl(String protocol, String host, int port, String u
                 password = (credentials.getStringValue(StringUtils.fromString(FtpConstants.ENDPOINT_CONFIG_PASS_KEY)))
                         .getValue();
             }
+        } else {
```

Comment:
Shall we update the default values of line #119 and #120 with these values? So that we can completely remove the `else` part here.

Reply:
Changed.